designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,352 @@
+    implementing this RFC as possible.
+-->
+
+## Backwards Compatibility Analysis
+
+HTTP redirects from existing content URL paths to the new content URL paths must be included as part of this project.
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+## Alternatives
+
+- Do not change the IA. Only update documentation within the existing IA as part of the [documentation update project](https://github.com/eslint/eslint/issues/16365).
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+## Open Questions
+
+1. Is there a way to automatically change all the internal links that'll be broken by these updates?
+   - A: [No (link with explanation)](https://github.com/eslint/rfcs/pull/97#discussion_r1003895871)
+1. Can you create a side navigation folder that is not also a page?
+   - A: [No (link with explanation)](https://github.com/eslint/rfcs/pull/97#discussion_r1003896032)
+1. How to make section landing pages navigable from the side table of contents? (For example, the proposed page "Use ESLint in Your Project landing page")
+   - [Cannot for top level landing pages for design decision reasons (link with explanation)](https://github.com/eslint/rfcs/pull/97#discussion_r1007459580)
+
+<!--
+    This section is optional, but is suggested for a first draft.
+
+    What parts of this proposal are you unclear about? What do you
+    need to know before you can finalize this RFC?
+
+    List the questions that you'd like reviewers to focus on. When
+    you've received the answers and updated the design to reflect them,
+    you can remove this section.
+-->
+
+## Help Needed
+
+I would likely need help with the following tasks:
+
+- Technical implementation details of how to handle redirects
+- I may need some help with updating the side navigation.
+- Investigate if there's a way to automate changing the internal link paths.
+
+<!--
+    This section is optional.
+
+    Are you able to implement this RFC on your own? If not, what kind
+    of help would you need from the team?
+-->
+
+## Frequently Asked Questions
+
+### Why Rename Pages with Active Verb Phrases?
+
+This outline proposes renaming various titles to use active verb phrases instead of the current gerund-based page name structure (ex. changing a page title from “Managing Issues” → “Manage Issues”). This is following a general technical writing best practice of avoiding gerunds in titles. For more information, refer to [this blog post about the subject](https://www.google.com/url?q=https://www.writethedocs.org/blog/newsletter-october-2022/%23gerunds-in-headings&sa=D&source=docs&ust=1665539063991039&usg=AOvVaw2HTT7RUQYPCcUenwVv0-Zt).
```

Comment:
Thanks for sharing this link! I was going to leave a comment at the top asking to learn more, but you anticipated that curiosity 🧠

Reply:
i had a feeling someone would comment 😃